arm-compute/2016-09-30/swagger/containerService.json

```diff
@@ -0,0 +1,595 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ContainerServiceClient",
+    "description": "The Container Service Client.",
+    "version": "2016-09-30"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
+  ],
+  "produces": [
+    "application/json",
+    "text/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerService/containerServices": {
+      "put": {
+        "tags": [
+          "ContainerServices"
+        ],
+        "operationId": "ContainerServices_List",
+        "description": "Lists all container services",
+        "parameters": {
+          "SubscriptionIdParameter": {
+            "ref": "#/parameters/SubscriptionIdParameter"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ContainerService"
```

Comment:
Shouldn't this be "ContainerServiceListResult" ? Is the service team testing the swagger spec before updating/sending the PR ?